portfolio/src/main/webapp/honorable_mentions.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```

Comment:
Clean looking HTML btw.  I'm curious if you manually indented these or if your IDE auto formats.

Reply:
It's a combination of both. I'm using the builtin editor on the cloudshell.